docs/source/api_docs_for_geeks.rst

```diff
@@ -10,94 +10,91 @@ auto_ml
-
-  :rtype: None. This is purely to fit the entire pipeline to the data. It doesn't return anything- it saves the fitted pipeline as a property of the ``Predictor`` instance.
+.. py:method:: ml_predictor.train(raw_training_data, user_input_func=None, optimize_final_model=False, perform_feature_selection=None, verbose=True, ml_for_analytics=True, take_log_of_y=None, model_names='GradientBoosting', perform_feature_scaling=True, calibrate_final_model=False, verify_features=False, cv=2, feature_learning=False, fl_data=None, prediction_intervals=False)
 
   :param raw_training_data: The data to train on. See below for more information on formatting of this data.
   :type raw_training_data: DataFrame, or a list of dictionaries, where each dictionary represents a row of data. Each row should have both the training features, and the output value we are trying to predict.
 
   :param user_input_func: [default- None] A function that you can define that will be called as the first step in the pipeline, for both training and predictions. The function will be passed the entire X dataset. The function must not alter the order or length of the X dataset, and must return the entire X dataset. You can perform any feature engineering you would like in this function. Using this function ensures that you perform the same feature engineering for both training and prediction. For more information, please consult the docs for scikit-learn's ``FunctionTransformer``.
   :type user_input_func: function
 
-  :param ml_for_analytics: [default- True] Whether or not to print out results for which features the trained model found useful. If ``True``, auto_ml will print results that an analyst might find interesting.
-  :type ml_for_analytics: Boolean
-
   :param optimize_final_model: [default- False] Whether or not to perform GridSearchCV on the final model. True increases computation time significantly, but will likely increase accuracy.
   :type optimize_final_model: Boolean
 
-  :param perform_feature_selection: [default- True for large datasets, False for small datasets] Whether or not to run feature selection before training the final model. Feature selection means picking only the most useful features, so we don't confuse the model with too much useless noise. Feature selection typically speeds up computation time by reducing the dimensionality of our dataset, and tends to combat overfitting as well.
+  :param perform_feature_selection: [default- True for large datasets (&ge; 100,000 rows), False for small datasets] Whether or not to run feature selection before training the final model. Feature selection means picking only the most useful features, so we don't confuse the model with too much useless noise. Feature selection typically speeds up computation time by reducing the dimensionality of our dataset, and tends to combat overfitting as well.
```

Comment:
```&ge;``` should be html code for "greater than equal to" symbol [http://turner.faculty.swau.edu/webstuff/htmlsymbols.html](http://turner.faculty.swau.edu/webstuff/htmlsymbols.html)